test/multiply_test.cc

```diff
@@ -1,7 +1,8 @@
-#include "avx512_gemm.h"
+//#include "avx512_gemm.h"
```

Comment:
?

Reply:
The *avx512_gemm.h* include should be taken care by intgemm.h.

Also all other gemm includes should be removed